etc/profile-a-l/cawbird.profile

```diff
@@ -0,0 +1,45 @@
+# Firejail profile for cawbird
+# Description: Open-source Twitter client for Linux
+# This file is overwritten after every install/update
+# Persistent local customizations
+include cawbird.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/cawbird
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-programs.inc
+include disable-passwdmgr.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+apparmor
+caps.drop all
+machine-id
+netfilter
+nodvd
+no3d
+nogroups
+nonewprivs
+noroot
+nosound
```

Comment:
Can it play videos?
---
Yes, please rebase.

Reply:
I've tested it with two gifs and one video from WHO,I don't get a lot of videos on my timeline. No sound though, do we need sound and hyperlinking? I'll close this for now and rebase and open another PR later if that's okay.

Reply:
> I'll close this for now and rebase and open another PR later if that's okay.

You can also force push.

> do we need sound

If sound works w/o FJ and breaks with, we will get new issues.

Reply:
And hyperlinking? Cawbird has some API restrictions,like polls/group DMs for example,not that I use those...

Reply:
If it is not such uncommon, we maybe should add a note which can be uncommented.